sqldb/paginate.go

```diff
@@ -6,20 +6,19 @@ import (
 )
 
 // QueryConfig holds configuration values for SQL queries.
+//
+//nolint:ll
 type QueryConfig struct {
 	// MaxBatchSize is the maximum number of items included in a batch
 	// query IN clauses list.
-	MaxBatchSize int
+	MaxBatchSize int `long:"max-batch-size" description:"The maximum number of items to include in a batch query IN clause. This is used for queries that fetch results based on a list of identifiers."`
```

Comment:
nit: could use uint32 here and below too.

Reply:
sure - will add a commit to [the follow up](https://github.com/lightningnetwork/lnd/pull/10154) PR if that's ok

Reply:
added a commit there: https://github.com/lightningnetwork/lnd/pull/10154/commits/f560c4d95b0127f569813b91fca949dae8461d94